.applier/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,65 @@
+---
+namespace: pipelinelib-testing
+repository_url: https://github.com/redhat-cop/pipeline-library.git
+slave_repo_ref: master
+templates_repo_ref: v1.4.13
+
+secrets:
+  dockercfg_json: '{"auths": { "{{ internal_registry_url.split("/")[0] }}": {"username": "openshift1", "password": "{{ oc_token }}", "email": "unused"}}}'
+
+jenkins:
+  deploy:
+    MEMORY_REQUEST: 2Gi
+    MEMORY_LIMIT: 3Gi
+    CPU_REQUEST: 2
+    CPU_LIMIT: 3
+  secrets:
+    OC_TOKEN_B64: "{{ oc_token | b64encode }}"
+    REGISTRY_URL_B64: "{{ internal_registry_url.split('/')[0] | b64encode }}"
+    DOCKERCFG_B64: "{{ secrets.dockercfg_json | to_json | b64encode }}"
+  tests:
+    files: "{{ lookup('fileglob', '{{ inventory_dir }}/../test/Jenkinsfile-*', wantlist=true) }}"
+
+openshift_cluster_content:
+- object: Environment Setup
+  content:
+    - name: Create Projects
```

Comment:
I know yaml isn't forcing us to use indentation for lists but lets pick one them.